documentation/modules/creating-migration-plan-2-8.adoc

```diff
@@ -39,18 +39,13 @@ The details of the plan are listed, and you can edit the items you filled in on
 
 ifdef::vmware[]
 
-* *Warm migration*: By default, all migrations are cold migrations. For a warm migration, click the *Edit* icon and select *Warm migration*.
+* *Migration type*: Type of migration. By default, {project-short} sets the migration type to `cold`. For a warm migration, click the *Edit* icon and select *warm*.
```

Comment:
lgtm.
Please note that this is the popup displayed when clicking the "Edit" icon
Maybe worth adding few words about cold/warm migrations here? unless it's displayed elsewhere in doc

![Screenshot from 2025-03-03 21-21-54](https://github.com/user-attachments/assets/19445539-452d-49dc-9a15-33d79b6e85a2)

Reply:
@anarnold97 Please review this PR. If good, please hold until it's time to merge for 2.8.0. Thanks.